index.php

```diff
@@ -9,13 +10,8 @@
 
 
 $serveAPI = new App\HandlingResources();
-$suppliers = new App\Suppliers(['https://bit.ly/3GlDsSw', 'https://bit.ly/3Gr5T1t']);
-
-
-foreach ($suppliers->suppliers as $supplier) {
-  $serveAPI->init_arr = [...$serveAPI->init_arr, ...json_decode(file_get_contents($supplier))];
-}
 
+$serveAPI->data = $serveAPI->collect($suppliers);
 $serveAPI->data = $serveAPI->format($serveAPI->init_arr, $serveAPI->data);
 $serveAPI->data = $serveAPI->sort($serveAPI->data);

```

Comment:
$serveAPI->data 
in here you are trying to set public properties in the class  "handleResource".
This is not a good practice.
so I suggest 2 solutions, pick what you feel good
1- use private properties with getter and setter
2- move the whole part to "handleResource" and just expose one public method to be called from index.php